pytest.ini

```diff
@@ -2,6 +2,10 @@
 log_level = DEBUG
 log_file = tests/.pytest.log
 log_file_level = DEBUG
+
+# Sets the parallelism of the tests that are curretnly running.
+addopts = -nauto
```

Comment:
How many cores does this use?

Reply:
It uses a default number that makes sense based on your hardware, basically number of cores (x2 if you have hyperthreading)

Reply:
Cool